src/libs/actions/Report.js

```diff
@@ -531,32 +531,22 @@ function getSimplifiedIOUReport(reportData) {
 /**
  * Fetches the updated data for an IOU Report and updates the IOU collection in Onyx
  *
+ * @param {Number} reportID
  * @param {Object[]} reportHistory
  */
-function updateIOUReportData(reportHistory) {
+function updateIOUReportData(reportID, reportHistory) {
     const containsIOUAction = _.any(reportHistory, action => action.actionName === 'IOU');
 
     // If there aren't any IOU actions, we don't need to fetch any additional data
     if (!containsIOUAction) {
         return;
     }
 
-    const otherParticipants = _.chain(reportHistory)
-        .pluck('actorEmail')
-        .unique()
-        .without(currentUserEmail)
-        .value();
-
-    // If we have more than one participant, this is not an IOU
-    if (otherParticipants.length > 1) {
-        return;
-    }
-
     // Since the Chat and the IOU are different reports with different reportIDs, and GetIOUReport only returns the
     // IOU's reportID, keep track of the IOU's reportID so we can use it to get the IOUReport data via `GetReportStuff`
     let iouReportID = 0;
     API.GetIOUReport({
-        debtorEmail: otherParticipants[0],
```

Comment:
I think there needs to be a little bit more logic below in the `.then()` to handle the case where there is a reportID of 0 returned (which means we don't need to get reportStuff). I also think it should probably have a couple of `jsonCode === 200` checks as well since these report APIs can throw all kinds of errors that we aren't always prepared for.